src/index.html

```diff
@@ -7,8 +7,37 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href = "https://fonts.googleapis.com/css?family=Roboto+Mono|Roboto+Slab|Roboto:300,400,500,700" rel = "stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div>
+        <a href="#home">
+          <img src="/images/logo.png"
+          alt="logo"
+          class="header_logo"
+          >
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="link_active"><a href="#1" class="is-active">Apple</a></li>
```

Comment:
```suggestion
          <li class="link_active">
            <a href="#1" class="is-active">Apple</a>
          </li>
```
Новый тег с новой строки